vendor/blanket-require.js

```diff
@@ -89,18 +90,24 @@ if (blanket.options('enableCoverage')) {
  * entries and annotate them so blanket will indicate their non-coverage
  */
 moduleLoaderFinish = function() {
-  if (blanket.options('enableCoverage')) {
-    for (var moduleName in requirejs.entries) {
-      if (typeof(seen[moduleName]) === 'undefined') {
-        seen[moduleName] = true;
-        if (!shouldExclude(moduleName)) {
-          try {
-            blanketLoader(moduleName);
-          } catch (err) {
-            console.log(err);
-          }
-        }
-      }
-    }
-  }
+  // if (blanket.options('enableCoverage')) {
```

Comment:
@sglanzer not sure if this is required anymore - can't find a suitable test case for it

in any case, as written it will no longer work (and suffers from the not everything is in requirejs.entries issue)